### PR TITLE
SAK-43947 ASN_ASSIGNMENT_PROPERTIES.VALUE conversion

### PR DESCRIPTION
**Pull request by ern**
Please add the Oracle conversion to this PR

### CODE REVIEW

**Comment by bjones86** [comment]
Oracle's max size for `VARCHAR2` is 4000 characters (I think this is extendable in newer versions of Oracle, but requires DBA configuration). Equivalent standard Oracle would be:

```
ALTER TABLE ASN_ASSIGNMENT_PROPERTIES MODIFY ( VALUE VARCHAR2(4000) NULL );
```

**Comment by davidpbauer** [comment]
Modify won't work in Oracle going from clob to varchar2 if there is data. You need something like this:
`alter table asn_assignment_properties add temp_value VARCHAR2(4000) NULL;
update asn_assignment_properties set temp_value=dbms_lob.substr(value,4000,1);
alter table asn_assignment_properties drop column value;
alter table asn_assignment_properties rename column temp_value to value;`

That will keep only the first 4000 characters of the value column. Alternatively you could have it throw an error to address the records that are too long.

**Comment by bjones86** [comment]
Nice, good call @davidpbauer, I overlooked that aspect. Thanks!

File: docs/conversion/sakai_20_1-20_2_mysql_conversion.sql [hunk 1]
```diff
@@ -2,3 +2,7 @@
 UPDATE poll_option SET deleted = 0 WHERE DELETED IS NULL;
 ALTER TABLE poll_option MODIFY COLUMN DELETED BIT NOT NULL DEFAULT 0;
 -- End SAK-44420
+
+-- SAK-43497
+alter table ASN_ASSIGNMENT_PROPERTIES modify VALUE varchar(4096) null;
```
Comment (ern) [inline docs/conversion/sakai_20_1-20_2_mysql_conversion.sql:7]:
```suggestion
alter table ASN_ASSIGNMENT_PROPERTIES modify VALUE varchar(4000) null;
```

**Comment by ern** [comment]
Updated conversion script for oracle

**Pull request merged**